apps/src/blockly/addons/cdoFieldVariable.js

```diff
@@ -50,6 +50,34 @@ export default class CdoFieldVariable extends GoogleBlockly.FieldVariable {
       }
     }
   }
+
+  /**
+   * Override of createTextArrow_ to fix the arrow position on Safari.
+   * We need to add dominant-baseline="central" to the arrow element in order to
+   * center it on Safari.
+   *  @override */
+  createTextArrow_() {
```

Comment:
I questioned whether we need two copies of this, but I think that we do. Blockly's `FieldVariable` class extends `FieldDropdown`.
`CdoFieldVariable` extends `FieldVariable` but not `CdoFieldDropdown`. Is that the right reasoning?

Reply:
Yep, we want `CdoFieldVariable` to extend `FieldVariable`, which we can't make extend our version of `FieldDropdown`.